internal/transformer/classes/analysers/compose/v3.go

```diff
@@ -158,7 +158,7 @@ func (c *V3Loader) convertToIR(filedir string, composeObject types.Config, servi
 		serviceContainer.Command = composeServiceConfig.Entrypoint
 		serviceContainer.Args = composeServiceConfig.Command
 		serviceContainer.Stdin = composeServiceConfig.StdinOpen
-		serviceContainer.Name = strings.ToLower(composeServiceConfig.ContainerName)
+		serviceContainer.Name = common.NormalizeForServiceName(composeServiceConfig.ContainerName)
```

Comment:
Do we want to create a new function for normalizing container names? It might have different requirements? Also, does it need to do ToLower?

Reply:
I am using an existing function common.NormalizeForServiceName which does all this. Container names are lowercase by default. I remember getting error from docker cli when camel-case was used.